source/common/stats/symbol_table_impl.h

```diff
@@ -594,7 +594,7 @@ class StatNameStorageSet {
 public:
   using HashSet =
       absl::flat_hash_set<StatNameStorage, HeterogeneousStatNameHash, HeterogeneousStatNameEqual>;
-  using iterator = HashSet::iterator;
+  using Iterator = HashSet::iterator;
```

Comment:
I'm was on the fence about exempting this since `::iterator` is more STL-like, but prefer fewer exemptions since it doesn't _have_ to be `lower_case` (unlike `is_transparent` and `type`) - open to opinions.